tests/unit/test_schema.py

```diff
@@ -80,6 +86,8 @@ def test_constructor_explicit(self):
                 )
             ),
         )
+        self.assertEqual(field.rounding_mode, ROUNDINGMODE.name)
```

Comment:
Nit: The string `"ROUNDING_MODE_UNSPECIFIED"` would be easier to convince myself that the code is correct. With `.name`, this is a bit of a "change detector test", since we've duplicated that logic from the implementation.

Reply:
Done.